src/validations/rules/ssp.sch

```diff
@@ -696,15 +697,19 @@
                 value="substring-before(@control-id, '-')" />
             <sch:let
                 name="leveragedUUID"
-                value="prop[@name = 'leveraged-authorization-uuid']/@value" />
+                value="oscal:prop[@name = 'leveraged-authorization-uuid']/@value" />
             <sch:assert
                 diagnostics="leveraged-PE-controls-implemented-requirement-diagnostic"
                 id="leveraged-PE-controls-implemented-requirement"
                 role="warning"
                 test="
-                    if ($leveraged/@uuid eq $leveragedUUID and $familyName eq 'pe')
+                    if ($familyName eq 'pe')
                     then
-                        false()
+                        if ($leveraged/@uuid eq $leveragedUUID)
```

Comment:
There is an additional complication: there can be more than one leveraged authorization (defined at line 692), thus the `eq` must be a set membership test using `=`.

I'm also having a difficult time finding an example in the SSP guide with the `oscal:implemented-requirement/oscal:prop[@name = 'leveraged-authorization-uuid']`

Reply:
equality is changed.

leveraged-authorization-uuid is in section 5.3.1.1 in the paragraph starting 'if the control origination is inherited, '  Also shown in the code example section, first prop child of implemented-requirement.

Reply:
I found an additional problem: https://github.com/GSA/fedramp-automation-guides/issues/21.

The definition of the variable `leveraged` (line 692) should be `//oscal:system-implementation/oscal:leveraged-authorization"`.

Reply:
Fixed